docker-compose-parts/api.devel.yml

```
@@ -18,3 +18,4 @@ services:
       target: production
     env_file:
       - .env
+    command: sleep 99999
```

I don't like this sleep but if it is needed will be better use infinite.
```suggestion
    command: sleep infinity
```